velox/expression/CastExpr-inl.h

```diff
@@ -30,10 +30,10 @@ inline std::string makeErrorMessage(
     const TypePtr& toType,
     const std::string& details = "") {
   return fmt::format(
-      "Failed to cast from {} to {}: {}. {}",
+      "Cannot cast {} '{}' to {}. {}",
```

Comment:
I find the cast fail message is different with Presto, do I need to change it in a separate PR or use the old error message? @mbasmanova

Reply:
Same PR should be fine. Thank you for noticing and fixing.